### PR TITLE
Add YARP_telemetry to the Dynamics profile

### PR DESCRIPTION
**Pull request by Nicogene**
As per the title.

### CODE REVIEW

File: cmake/BuildYARP_telemetry.cmake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+              REPOSITORY robotology/yarp-telemetry.git
+              TAG master
+              COMPONENT dynamics
+              FOLDER src
+              DEPENDS YCM
+                      YARP
+                      matioCpp)
+
+set(YARP_telemetry_CONDA_DEPENDENCIES boost-cpp)
+set(YARP_telemetry_CONDA_VERSION "2.3.0.1")
```
Comment (traversaro) [inline cmake/BuildYARP_telemetry.cmake:22]: This is an override that should not be necessary if the correct version is added in latest releases file, did you added it for a specific reason or it is just a copy&paste leftover?

Comment (Nicogene) [reply cmake/BuildYARP_telemetry.cmake:22]: Do you mean line 22?
Actually, it is a copy-paste and I wanted to ask you what those variables are fore 😄

Comment (traversaro) [reply cmake/BuildYARP_telemetry.cmake:22]: A copy-paste from where?

Comment (traversaro) [reply cmake/BuildYARP_telemetry.cmake:22]: Some docs are available in https://github.com/robotology/robotology-superbuild/blob/master/doc/developers-faqs.md#how-to-ensure-that-binary-packages-are-correctly-generated-for-a-new-package, but probably the docs on version are missing.

Comment (Nicogene) [reply cmake/BuildYARP_telemetry.cmake:22]: >A copy-paste from where?

From `blocktestcore`

Comment (traversaro) [reply cmake/BuildYARP_telemetry.cmake]: Good catch, indeed the version number is wrong: https://anaconda.org/robotology/blocktest .

**Comment by traversaro** [comment]
@Nicogene I bumped the used version of robotology/robotology-superbuild-dependencies-vcpkg in https://github.com/robotology/robotology-superbuild/pull/677/commits/c02873ed6b9a49d75748a9aba693b4de30532438 . Can you update the CHANGELOG describing this change? Thanks!

**Review by traversaro — APPROVED** [review]

**Comment by traversaro** [comment]
Thanks @Nicogene and @AlexAntn !

**Pull request merged**